bin/common/create_kubedns.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+echo "Waiting for kubectl to respond..."
+while ! kubectl get pods --all-namespaces; do
+  sleep 2
+done
+echo "Waiting for kube-apiserver to be active..."
+while ! systemctl is-active kube-apiserver.service 2>/dev/null >/dev/null; do
+  sleep 10
+done
+# Due to timing, kube-system may not exist immediately
+while ! kubectl get ns kube-system >& /dev/null; do
```

Comment:
Oh, didn't realize `>&` worked too. As opposed to writing to a file named `&` (which it doesn't do).  Nevermind then!